base/Dockerfile

```diff
@@ -29,5 +29,7 @@ RUN gem2.0 install aws-sdk-resources --pre
 # Credential management bits
 COPY clortho-get /etc/pre-init.d/clortho-get
 
+#Create Data Directory
+ENV METRICS_ROOT_DIR /data/metrics
```

Comment:
I like placing metrics under the data directory.  This is also the common directory where we mount all our persistent volumes.  It will make it easy for us to use more reliably persistence in the future if we so choose to.